addons/analytic/models/analytic_account.py

```diff
@@ -14,14 +14,23 @@ class AccountAnalyticDistribution(models.Model):
 
     account_id = fields.Many2one('account.analytic.account', string='Analytic Account', required=True)
     percentage = fields.Float(string='Percentage', required=True, default=100.0)
-    name = fields.Char(string='Name', related='account_id.name', readonly=False)
+    name = fields.Char(string='Name', readonly=True, compute="_compute_name")
```

Comment:
name_get?

Reply:
What's the tag below for?

Reply:
> What's the tag below for?

It was just the counterpart of the One2many of the tanalytic.tag, but as I will delete the tags, I will delete this field too.